src/Receipt/DonationDetailsGroup/Details/PaymentGateway.php

```diff
@@ -0,0 +1,28 @@
+<?php
+namespace Give\Receipt\DonationDetailsGroup\Details;
+
+use Give\Receipt\Detail;
+use function give_get_payment_meta as getDonationMetaData;
+use function give_get_gateway_admin_label as getGatewayLabel;
+
+/**
+ * Class PaymentGateway
+ *
+ * @since 2.7.0
+ * @package Give\Receipt\DonationDetailsGroup\Details
+ */
+class PaymentGateway extends Detail {
+	/**
+	 * @inheritDoc
+	 */
+	public function getLabel() {
+		return __( 'PAYMENT METHOD', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.